plugins/orchestrator-backend/package.json

```
@@ -92,5 +99,10 @@
     "@types/express": "4.17.20",
     "@types/fs-extra": "11.0.1",
     "@types/json-schema": "7.0.12"
-  }
+  },
+  "maintainers": [
+    "@janus-idp/maintainers-plugins",
+    "@janus-idp/orchestrator-codeowners"
+  ],
+  "author": "'The"
```

```suggestion
  "author": "Red Hat"
```
?
There are other packages with the same value.

---

odd. I'll look into what the heck happened there. It should definitely be `Red Hat` unless the plugin is owned/contrib'd by someone outside RH.

---

fixed in https://github.com/janus-idp/backstage-plugins/pull/1627/commits/ae211d2c40c5aca045c335b1cee5e579f1ec55d2#diff-e3fb4fd157240487ad11e13af78c9bf58c055218b2f3ce9dbd897201fa585f83R12 ... turns out there's value to all the bash linting reccos I was ignoring :D

---

@caponetto  Would you prefer if these 4 plugins were authored by RH ?

```
$➔ for d in plugins/orchestrator*/package.json; do echo; echo $d; jq .author $d; done

plugins/orchestrator-backend/package.json
"The Backstage Community"

plugins/orchestrator-common/package.json
"The Backstage Community"

plugins/orchestrator/package.json
"The Backstage Community"

plugins/orchestrator-swf-editor-envelope/package.json
"The Backstage Community"
```

---

@pkliczewski do you mind answering these questions, please? ^